sky/server/requests/queues/local_queue.py

```diff
@@ -0,0 +1,16 @@
+"""Process-local queue implementation."""
```

Comment:
as mentinoed earlier, do we need a separate implementation of the queue for local mode. Would it be better if we just consolidate the two implementation to use the DB as a queue, which might be easier to scale as well?

Reply:
Just a quick fix to avoid the queue sever process in low resources env, I think an external queue like sqlite is a must-have regarding failover and scaling issues. My concern is adding it in this PR might explode the scope. We can safely remove the `LOCAL` and `MULTIPROCESSING` queue implementation after we adopt the DB solution since they are stateless between restart.